test/IECoreMaya/ToMayaPlugConverterTest.py

```diff
@@ -37,19 +37,67 @@
 import IECore
 import IECoreMaya
 
+
 class ToMayaPlugConverterTest( IECoreMaya.TestCase ) :
 
 	def testSinglePlugConversion( self ) :
 
 		locator = maya.cmds.spaceLocator()[0]
 
-		converter = IECoreMaya.ToMayaPlugConverter.create( IECore.FloatData( 10 ) )
+		converter = IECoreMaya.ToMayaPlugConverter.create( IECore.DoubleData( 10 ) )
 		self.assert_( converter.isInstanceOf( IECoreMaya.ToMayaPlugConverter.staticTypeId() ) )
 		self.assert_( converter.isInstanceOf( IECoreMaya.ToMayaConverter.staticTypeId() ) )
 		self.assert_( converter.isInstanceOf( IECore.FromCoreConverter.staticTypeId() ) )
 
 		converter.convert( locator + ".translateX" )
-		self.assertEqual( maya.cmds.getAttr( locator + ".translateX" ), 10 )
+		self.assertAlmostEqual( maya.cmds.getAttr( locator + ".translateX" ), 10 )
+
+	def testUnitPlugConversion(self):
+
+		timeNode = maya.cmds.createNode('time')
+		maya.cmds.currentUnit( time='film' )  # set to 24fps
+		converter = IECoreMaya.ToMayaPlugConverter.create( IECore.DoubleData( 1.23 ) )
+		converter.convert(timeNode + '.outTime')
+		self.assertAlmostEqual(maya.cmds.getAttr(timeNode + '.outTime'), 1.23 * 24.0)
+
+		locator = maya.cmds.spaceLocator()[0]
+		converter = IECoreMaya.ToMayaPlugConverter.create( IECore.DoubleData( IECore.degreesToRadians(90.0) ) )
+		converter.convert(locator + '.rotateX' )
+		self.assertAlmostEqual(maya.cmds.getAttr(locator + '.rotateX'), 90.0)
+
+		converter = IECoreMaya.ToMayaPlugConverter.create( IECore.DoubleData( 1.23456) )
+		converter.convert(locator + '.translateX' )
+		self.assertAlmostEqual(maya.cmds.getAttr(locator + '.translateX'), 1.23456)
+
+	def testNumericPlugConversion(self):
+
+		locator = maya.cmds.spaceLocator()[0]
+
+		testCases = [
```

Comment:
Perhaps this is just me but I prefer a straight line function for tests, even cutting and pasting code if necessary. For example here you wouldn't need the special cases in the setup & checks.

I'm not suggesting you change this test now but perhaps @andrewkaufman can adjudicate on future style.